xds/src/test/java/io/grpc/xds/ClientLoadCounterTest.java

```diff
@@ -194,7 +194,7 @@ public void metricsRecordingListener_backendMetricsAggregation() {
         OrcaLoadReport.newBuilder()
             .setCpuUtilization(0.3423)
             .setMemUtilization(0.654)
-            .putRequestCostOrUtilization("named-cost-or-utilization", 3534.0)
+            .putUtilization("named-utilization", 0.7563)
```

Comment:
Where is 0.7563 coming from? Can you add a comment?

Reply:
This is an arbitrary number chosen for test.

Reply:
Why it has to change?

Reply:
Previously, the map was `request_cost_or_utilization`, which contains both cost (i.e., absolute cost and its value can be arbitrary) and utilization (i.e., a fraction of total resources available and its value is constrained to be in range [0, 1]).

Reply:
I see.